src/Platform_Core/org/lobobrowser/request/ClientletRequestHandler.java

```diff
@@ -97,7 +98,7 @@ public boolean handleException(final ClientletResponse response, final Throwable
    * .ClientletResponse)
    */
   @Override
-  public void processResponse(final ClientletResponse response) throws ClientletException, IOException {
+  public void processResponse(final ClientletResponse response, Consumer<Boolean> consumer) throws ClientletException, IOException {
```

Comment:
In all these other `processResponse` implementations, the response is "valid", so you will have to call `consumer.accept(true)` somewhere in the function (most likely at the end of the function).